src/styles/global.css

```diff
@@ -0,0 +1,24 @@
+@import "/src/assets/fonts/inter.css";
+@import "/src/styles/ui-colors.css";
+
+:root {
+    --max-width: 100%;
+    --border-radius: 12px;
+    --font: Inter;
+}
+
+* {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+}
+
+html,
+body {
+    max-width: 100vw;
+    overflow-x: hidden;
```

Comment:
this rule should not be needed and if it is needed, we should find the root of this issue somewhere else in your css

Reply:
you resolved this, but it is not resolved